nwb.config.js

```diff
@@ -1,5 +1,10 @@
+process.env.CHROME_BIN = require('puppeteer').executablePath()
+
 module.exports = {
   type: 'react-component',
+  karma: {
+    browsers: ["ChromeHeadless"]
```

Comment:
what is ChromeHeadless !

Reply:
https://www.npmjs.com/package/karma-chrome-launcher